src/Components/Activities/Activities.js

```diff
@@ -27,11 +27,11 @@ const Activities = ({ activities, updateActivity, setFeeling }) => {
   return (
     <section className="activity-page">
       <div className="activity-card">
-        {currentActivityIndex > 0 && <button onClick={prevActivity}>←</button>}
+        {currentActivityIndex > 0 && <button className="activities-btns" onClick={prevActivity}>←</button>}
         <h2>{activities[currentActivityIndex]?.activity}</h2>
-        {currentActivityIndex < activities.length - 1 && <button onClick={nextActivity}>→</button>}
+        {currentActivityIndex < activities.length - 1 && <button className="activities-btns" onClick={nextActivity}>→</button>}
       </div>
-      <Link to="/why-are-you-feeling-that-way" className="go-button" onClick={selectActivity}>Let's do it!</Link>
+      <Link to="/why-are-you-feeling-that-way" className="uni-btn" onClick={selectActivity}>Let's do it!</Link>
```

Comment:
Love that the size stays consistent!!

My only comment here is that I'd like to see the buttons stay in one place on each side, as opposed to shifting position with the width of the phrase. It makes it hard to click through efficiently.

Reply:
10000% Thanks for pointing that out

Reply:
+++++ Joe